src/components/rangeInput.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { FC } from "react"
+
+import Input from "./input/index"
+
+import Label from "./fieldHelpers/label"
+
+interface Props {
+  name: {
+    min: string
+    max: string
+  }
```

Comment:
Couldn't we pass a single name and construct `min` and `max` names with an appropriate suffix?

Reply:
Yes 👍